asf_heat_pump_suitability/analysis/flats_on_fossils/analysis_fill_building_rise_methods.py

```diff
@@ -0,0 +1,629 @@
+# %% [markdown]
+# # Test different methodologies to fill building rise/ storey count information using building height
+#
+# FLAT_STOREY_COUNT data is available from EPC data and building height estimates are available from Microsoft ML Global Building Footprints dataset.
+#
+# FLAT_STOREY_COUNT has low coverage of non-null values in EPC and Microsoft ML Global Building Footprints has decent coverage. Here we test 4 different basic methodologies for converting building height data to FLAT STOREY COUNT data. We compare the methodologies at the end to select the best one.
+
+# %%
+import polars as pl
+import numpy as np
+import matplotlib.pyplot as plt
+import sklearn
```

Comment:
The LinearRegression didn't work by accessing this object, I had to import like the following:

```suggestion
import sklearn
from sklearn.linear_model import LinearRegression
```

Reply:
oh! i will test to make sure it works for me and then update. thanks!